sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -4601,6 +4601,257 @@ case class ArrayExcept(left: Expression, right: Expression) extends ArrayBinaryL
     newLeft: Expression, newRight: Expression): ArrayExcept = copy(left = newLeft, right = newRight)
 }
 
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = "_FUNC_(x, pos, val) - Places val into index pos of array x (array indices start at 1, or start from the end if start is negative).\",",
```

Comment:
Can we document the behavior of `pos` is larger than the array length?

Reply:
@cloud-fan Apologies for the delay, something came up at work. Please see the doc fix here: https://github.com/apache/spark/pull/39975 (aligned to the python docstring also).